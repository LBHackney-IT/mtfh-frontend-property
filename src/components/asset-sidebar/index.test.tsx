import React from "react";

import { render, server } from "@hackney/mtfh-test-utils";
import { screen } from "@testing-library/react";
import { rest } from "msw";

import { AssetSideBar } from ".";

import { Asset } from "@mtfh/common/lib/api/asset/v1";
import * as auth from "@mtfh/common/lib/auth/auth";

const assetData: Asset = {
  id: "769894bd-b0bc-47eb-a780-322372c2448f",
  assetId: "0019062023",
  assetType: "Block",
  assetLocation: {
    parentAssets: [],
    floorNo: "0",
    totalBlockFloors: 3,
  },
  assetAddress: {
    uprn: "10008355547",
    addressLine1: "123 Test Block(CAN BE DELETED)",
    addressLine2: "",
    addressLine3: "",
    addressLine4: "",
    postPreamble: "",
    postCode: "FK8 1FH",
  },
  assetManagement: {
    isCouncilProperty: true,
    managingOrganisation: "London Borough of Hackney",
    managingOrganisationId: "c01e3146-e630-c2cd-e709-18ef57bf3724",
    isTMOManaged: false,
    agent: "Agent Smith",
    areaOfficeName: "Test",
    owner: "Test",
  },
  assetCharacteristics: {
    numberOfBedrooms: 0,
    numberOfLifts: 0,
    numberOfLivingRooms: 0,
    windowType: "Wood",
    yearConstructed: "1991",
    numberOfSingleBeds: null,
    numberOfDoubleBeds: null,
    numberOfFloors: null,
    totalBlockFloors: null,
    heating: null,
    propertyFactor: null,
    architecturalType: null,
  },
  versionNumber: 0,
  tenure: {
    id: "c01e3146-e630-c2cd-e709-18ef57bf3724",
    paymentReference: "",
    type: "",
    startOfTenureDate: "",
    endOfTenureDate: "",
    isActive: false,
  },
  rootAsset: "",
  parentAssetIds: "",
  rentGroup: null,
};

beforeEach(() => {
  jest.resetAllMocks();

  jest.spyOn(auth, "isAuthorisedForGroups").mockReturnValue(true);

  server.use(
    rest.get(
      `/api/v1/cautionary-alerts/properties-new/${assetData.assetId}`,
      (req, res, ctx) => res(ctx.status(200), ctx.json({ alerts: [] })),
    ),
  );
});

test("it shows cautionary alerts", () => {
  // Arrange
  const { container } = render(
    <AssetSideBar
      alerts={[]}
      assetDetails={assetData}
      showCautionaryAlerts
      showTenureInformation={false}
    />,
    {
      url: `/property/${assetData.id}`,
      path: "/property/:assetId",
    },
  );

  // Assert
  const cautionaryAlertsHeading = screen.getByText("Cautionary alerts");

  expect(cautionaryAlertsHeading).toBeVisible();

  expect(container).toMatchSnapshot();
});

test("it hides cautionary alerts", () => {
  // Arrange
  const { container } = render(
    <AssetSideBar
      alerts={[]}
      assetDetails={assetData}
      showCautionaryAlerts={false}
      showTenureInformation={false}
    />,
    {
      url: `/property/${assetData.id}`,
      path: "/property/:assetId",
    },
  );

  // Assert
  const cautionaryAlertsHeading = screen.queryByText("Cautionary alerts");

  expect(cautionaryAlertsHeading).not.toBeInTheDocument();

  expect(container).toMatchSnapshot();
});

test("it shows tenure information", () => {
  // Arrange
  const { container } = render(
    <AssetSideBar
      alerts={[]}
      assetDetails={assetData}
      showCautionaryAlerts={false}
      showTenureInformation
    />,
    {
      url: `/property/${assetData.id}`,
      path: "/property/:assetId",
    },
  );

  // Assert
  const tenureInformationHeading = screen.getByText("Tenure");

  expect(tenureInformationHeading).toBeVisible();

  expect(container).toMatchSnapshot();
});

test("it hides tenure information", () => {
  // Arrange
  const { container } = render(
    <AssetSideBar
      alerts={[]}
      assetDetails={assetData}
      showCautionaryAlerts={false}
      showTenureInformation={false}
    />,
    {
      url: `/property/${assetData.id}`,
      path: "/property/:assetId",
    },
  );

  // Assert
  const tenureInformationHeading = screen.queryByText("Tenure");

  expect(tenureInformationHeading).not.toBeInTheDocument();

  expect(container).toMatchSnapshot();
});
