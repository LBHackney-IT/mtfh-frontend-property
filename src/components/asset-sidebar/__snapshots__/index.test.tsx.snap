// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it hides boiler house details 1`] = `
<div>
  <div
    class="mtfh-asset-sidebar"
  >
    <div
      class="mtfh-sidebar"
    >
      <div
        id="property-view-sidebar"
      >
        <aside
          class="mtfh-asset-details"
        >
          <dl
            class="govuk-summary-list govuk-summary-list--no-border lbh-summary-list"
          >
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key govuk-!-width-two-thirds"
              >
                Type
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                Block
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                UPRN
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                10008355547
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                Reference
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                0019062023
              </dd>
            </div>
          </dl>
        </aside>
        <details
          class="govuk-details"
          data-module="govuk-details"
        >
          <summary
            class="govuk-details__summary"
            data-testid="property-spec-toggle"
          >
            <span
              class="govuk-details__summary-text"
            >
              Property Specification
            </span>
          </summary>
        </details>
        <a
          class="govuk-button lbh-button"
          data-testid="edit-address-button"
          href="/property/edit/769894bd-b0bc-47eb-a780-322372c2448f"
        >
          Edit address details
        </a>
        <hr
          class="lbh-horizontal-bar"
        />
        <aside
          class="mtfh-cautionary-alerts"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Cautionary alerts
          </h2>
          <p
            class="lbh-body-s"
          >
            None
          </p>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-patch-details"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Patch details
          </h2>
          <p
            class="lbh-body-s"
          >
            No patch
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="all-patches-and-areas-button"
            href="/property/all-patches-and-areas"
          >
            All patches and areas
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-lbh-ownership-info"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
            data-testid="ownership-info-heading"
          >
            Asset ownership
          </h2>
          <p
            class="lbh-body-s"
            data-testid="ownership-info"
          >
            This asset is owned by Hackney Council
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="edit-ownership-info-button"
            href="/property/769894bd-b0bc-47eb-a780-322372c2448f/edit-asset-ownership"
          >
            Edit ownership
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
      </div>
    </div>
  </div>
</div>
`;

exports[`it hides cautionary alerts 1`] = `
<div>
  <div
    class="mtfh-asset-sidebar"
  >
    <div
      class="mtfh-sidebar"
    >
      <div
        id="property-view-sidebar"
      >
        <aside
          class="mtfh-asset-details"
        >
          <dl
            class="govuk-summary-list govuk-summary-list--no-border lbh-summary-list"
          >
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key govuk-!-width-two-thirds"
              >
                Type
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                Block
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                UPRN
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                10008355547
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                Reference
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                0019062023
              </dd>
            </div>
          </dl>
        </aside>
        <details
          class="govuk-details"
          data-module="govuk-details"
        >
          <summary
            class="govuk-details__summary"
            data-testid="property-spec-toggle"
          >
            <span
              class="govuk-details__summary-text"
            >
              Property Specification
            </span>
          </summary>
        </details>
        <a
          class="govuk-button lbh-button"
          data-testid="edit-address-button"
          href="/property/edit/769894bd-b0bc-47eb-a780-322372c2448f"
        >
          Edit address details
        </a>
        <hr
          class="lbh-horizontal-bar"
        />
        <aside
          class="mtfh-patch-details"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Patch details
          </h2>
          <p
            class="lbh-body-s"
          >
            No patch
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="all-patches-and-areas-button"
            href="/property/all-patches-and-areas"
          >
            All patches and areas
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-lbh-ownership-info"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
            data-testid="ownership-info-heading"
          >
            Asset ownership
          </h2>
          <p
            class="lbh-body-s"
            data-testid="ownership-info"
          >
            This asset is owned by Hackney Council
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="edit-ownership-info-button"
            href="/property/769894bd-b0bc-47eb-a780-322372c2448f/edit-asset-ownership"
          >
            Edit ownership
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
      </div>
    </div>
  </div>
</div>
`;

exports[`it hides tenure information 1`] = `
<div>
  <div
    class="mtfh-asset-sidebar"
  >
    <div
      class="mtfh-sidebar"
    >
      <div
        id="property-view-sidebar"
      >
        <aside
          class="mtfh-asset-details"
        >
          <dl
            class="govuk-summary-list govuk-summary-list--no-border lbh-summary-list"
          >
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key govuk-!-width-two-thirds"
              >
                Type
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                Block
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                UPRN
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                10008355547
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                Reference
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                0019062023
              </dd>
            </div>
          </dl>
        </aside>
        <details
          class="govuk-details"
          data-module="govuk-details"
        >
          <summary
            class="govuk-details__summary"
            data-testid="property-spec-toggle"
          >
            <span
              class="govuk-details__summary-text"
            >
              Property Specification
            </span>
          </summary>
        </details>
        <a
          class="govuk-button lbh-button"
          data-testid="edit-address-button"
          href="/property/edit/769894bd-b0bc-47eb-a780-322372c2448f"
        >
          Edit address details
        </a>
        <hr
          class="lbh-horizontal-bar"
        />
        <aside
          class="mtfh-patch-details"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Patch details
          </h2>
          <p
            class="lbh-body-s"
          >
            No patch
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="all-patches-and-areas-button"
            href="/property/all-patches-and-areas"
          >
            All patches and areas
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-lbh-ownership-info"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
            data-testid="ownership-info-heading"
          >
            Asset ownership
          </h2>
          <p
            class="lbh-body-s"
            data-testid="ownership-info"
          >
            This asset is owned by Hackney Council
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="edit-ownership-info-button"
            href="/property/769894bd-b0bc-47eb-a780-322372c2448f/edit-asset-ownership"
          >
            Edit ownership
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
      </div>
    </div>
  </div>
</div>
`;

exports[`it shows boiler house details 1`] = `
<div>
  <div
    class="mtfh-asset-sidebar"
  >
    <div
      class="mtfh-sidebar"
    >
      <div
        id="property-view-sidebar"
      >
        <aside
          class="mtfh-asset-details"
        >
          <dl
            class="govuk-summary-list govuk-summary-list--no-border lbh-summary-list"
          >
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key govuk-!-width-two-thirds"
              >
                Type
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                Dwelling
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                UPRN
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                10008355547
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                Reference
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                0019062023
              </dd>
            </div>
          </dl>
        </aside>
        <details
          class="govuk-details"
          data-module="govuk-details"
        >
          <summary
            class="govuk-details__summary"
            data-testid="property-spec-toggle"
          >
            <span
              class="govuk-details__summary-text"
            >
              Property Specification
            </span>
          </summary>
        </details>
        <a
          class="govuk-button lbh-button"
          data-testid="edit-address-button"
          href="/property/edit/769894bd-b0bc-47eb-a780-322372c2448f"
        >
          Edit address details
        </a>
        <hr
          class="lbh-horizontal-bar"
        />
        <aside
          class="mtfh-cautionary-alerts"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Cautionary alerts
          </h2>
          <p
            class="lbh-body-s"
          >
            None
          </p>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-patch-details"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Patch details
          </h2>
          <p
            class="lbh-body-s"
          >
            No patch
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="all-patches-and-areas-button"
            href="/property/all-patches-and-areas"
          >
            All patches and areas
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-lbh-ownership-info"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
            data-testid="ownership-info-heading"
          >
            Asset ownership
          </h2>
          <p
            class="lbh-body-s"
            data-testid="ownership-info"
          >
            This asset is owned by Hackney Council
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="edit-ownership-info-button"
            href="/property/769894bd-b0bc-47eb-a780-322372c2448f/edit-asset-ownership"
          >
            Edit ownership
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-boiler-house"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Boiler house
          </h2>
          <a
            class="govuk-button lbh-button"
            href="/property/769894bd-b0bc-47eb-a780-322372c2448f/add-boiler-house"
          >
            Add boiler house 
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
      </div>
    </div>
  </div>
</div>
`;

exports[`it shows cautionary alerts 1`] = `
<div>
  <div
    class="mtfh-asset-sidebar"
  >
    <div
      class="mtfh-sidebar"
    >
      <div
        id="property-view-sidebar"
      >
        <aside
          class="mtfh-asset-details"
        >
          <dl
            class="govuk-summary-list govuk-summary-list--no-border lbh-summary-list"
          >
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key govuk-!-width-two-thirds"
              >
                Type
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                Block
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                UPRN
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                10008355547
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                Reference
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                0019062023
              </dd>
            </div>
          </dl>
        </aside>
        <details
          class="govuk-details"
          data-module="govuk-details"
        >
          <summary
            class="govuk-details__summary"
            data-testid="property-spec-toggle"
          >
            <span
              class="govuk-details__summary-text"
            >
              Property Specification
            </span>
          </summary>
        </details>
        <a
          class="govuk-button lbh-button"
          data-testid="edit-address-button"
          href="/property/edit/769894bd-b0bc-47eb-a780-322372c2448f"
        >
          Edit address details
        </a>
        <hr
          class="lbh-horizontal-bar"
        />
        <aside
          class="mtfh-cautionary-alerts"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Cautionary alerts
          </h2>
          <p
            class="lbh-body-s"
          >
            None
          </p>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-patch-details"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Patch details
          </h2>
          <p
            class="lbh-body-s"
          >
            No patch
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="all-patches-and-areas-button"
            href="/property/all-patches-and-areas"
          >
            All patches and areas
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-lbh-ownership-info"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
            data-testid="ownership-info-heading"
          >
            Asset ownership
          </h2>
          <p
            class="lbh-body-s"
            data-testid="ownership-info"
          >
            This asset is owned by Hackney Council
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="edit-ownership-info-button"
            href="/property/769894bd-b0bc-47eb-a780-322372c2448f/edit-asset-ownership"
          >
            Edit ownership
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
      </div>
    </div>
  </div>
</div>
`;

exports[`it shows tenure information 1`] = `
<div>
  <div
    class="mtfh-asset-sidebar"
  >
    <div
      class="mtfh-sidebar"
    >
      <div
        id="property-view-sidebar"
      >
        <aside
          class="mtfh-asset-details"
        >
          <dl
            class="govuk-summary-list govuk-summary-list--no-border lbh-summary-list"
          >
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key govuk-!-width-two-thirds"
              >
                Type
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                Block
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                UPRN
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                10008355547
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                Reference
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                0019062023
              </dd>
            </div>
          </dl>
        </aside>
        <details
          class="govuk-details"
          data-module="govuk-details"
        >
          <summary
            class="govuk-details__summary"
            data-testid="property-spec-toggle"
          >
            <span
              class="govuk-details__summary-text"
            >
              Property Specification
            </span>
          </summary>
        </details>
        <a
          class="govuk-button lbh-button"
          data-testid="edit-address-button"
          href="/property/edit/769894bd-b0bc-47eb-a780-322372c2448f"
        >
          Edit address details
        </a>
        <hr
          class="lbh-horizontal-bar"
        />
        <aside
          class="mtfh-patch-details"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            Patch details
          </h2>
          <p
            class="lbh-body-s"
          >
            No patch
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="all-patches-and-areas-button"
            href="/property/all-patches-and-areas"
          >
            All patches and areas
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-lbh-ownership-info"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
            data-testid="ownership-info-heading"
          >
            Asset ownership
          </h2>
          <p
            class="lbh-body-s"
            data-testid="ownership-info"
          >
            This asset is owned by Hackney Council
          </p>
          <a
            class="govuk-button lbh-button"
            data-testid="edit-ownership-info-button"
            href="/property/769894bd-b0bc-47eb-a780-322372c2448f/edit-asset-ownership"
          >
            Edit ownership
          </a>
          <hr
            class="lbh-horizontal-bar"
          />
        </aside>
        <aside
          class="mtfh-tenure-details"
          id="tenure-details"
        >
          <h2
            class="lbh-heading-h2 lbh-heading lbh-heading-h3"
          >
            <a
              class="govuk-link lbh-link lbh-link--no-visited-state"
              href="/tenure/c01e3146-e630-c2cd-e709-18ef57bf3724"
            >
              Tenure
            </a>
          </h2>
          <dl
            class="govuk-summary-list govuk-summary-list--no-border lbh-summary-list"
          >
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key govuk-!-width-two-thirds"
              >
                Type
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                N/A
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                Status
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                Inactive
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                Start date
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                N/A
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                End date
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                N/A
              </dd>
            </div>
            <div
              class="govuk-summary-list__row"
            >
              <dt
                class="govuk-summary-list__key"
              >
                Payment ref.
              </dt>
              <dd
                class="govuk-summary-list__value"
              >
                N/A
              </dd>
            </div>
          </dl>
        </aside>
        <a
          class="govuk-button lbh-button"
          href="/tenure/769894bd-b0bc-47eb-a780-322372c2448f/add"
        >
          New tenure
        </a>
      </div>
    </div>
  </div>
</div>
`;
